app/javascript/angular/code/controllers/_fixed_sessions_map_ctrl.js

```diff
@@ -85,7 +85,9 @@ export const FixedSessionsMapCtrl = (
       });
 
       elmApp.ports.findLocation.subscribe(location => {
-        FiltersUtils.findLocation(location, params, map);
+        FiltersUtils.findLocation(location, params, map, () => {
```

Comment:
Maybe we could use an object as an argument? Example
```
findLoc({
  location:
  params
  ...
```
Gets difficult to track arguments when it's that many